app/src/main/java/org/asteroidos/sync/connectivity/NotificationService.java

```diff
@@ -82,56 +83,61 @@ public final UUID getServiceUUID() {
         return AsteroidUUIDS.NOTIFICATION_SERVICE_UUID;
     }
 
+    @Override
+    public void postNotification(Context context, Intent intent) {
+        String event = intent.getStringExtra("event");
+        if (Objects.equals(event, "posted")) {
+            String packageName = intent.getStringExtra("packageName");
+            NotificationPreferences.putPackageToSeen(context, packageName);
+            NotificationPreferences.NotificationOption notificationOption =
+                    NotificationPreferences.getNotificationPreferenceForApp(context, packageName);
+            if (notificationOption == NotificationPreferences.NotificationOption.NO_NOTIFICATIONS)
+                return;
+
+            int id = intent.getIntExtra("id", 0);
```

Comment:
I guess this should be key now. One more reason to share code between the two implementations